SegmentEditorSurfaceCut/SegmentEditorSurfaceCutLib/SegmentEditorEffect.py

```diff
@@ -175,6 +192,17 @@ def onOperationSelectionChanged(self, operationName, toggle):
       return
     self.scriptedEffect.setParameter("Operation", operationName)
 
+  def onSmoothModelCheckboxStateChanged(self, newState):
+    logging.info('onSmoothModelCheckboxStateChanged, new state is: %s'%(str(newState)))
+    if newState==qt.Qt.Checked:
+      self.scriptedEffect.setParameter("SmoothModel", 'True')

```

Comment:
This is correct, but for consistency with how we save Boolean values, save it as 0/1 instead see this example:

https://github.com/Slicer/Slicer/blob/62db2030cba19974d8e0cd556c084a9e00049f4f/Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorLogicalEffect.py#L151-L152